src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -1017,21 +1026,31 @@ def _get_authorization_code_worker(tenant, aad_endpoint, resource, results):
         return
 
     # launch browser:
-    url = ('{0}/{1}/oauth2/authorize?response_type=code&client_id={2}'
-           '&redirect_uri={3}&state={4}&resource={5}&prompt=select_account')
-    url = url.format(aad_endpoint, tenant or _COMMON_TENANT, _CLIENT_ID, reply_url, 'code', resource)
+    url = ('{0}/oauth2/authorize?response_type=code&client_id={1}'
+           '&redirect_uri={2}&state={3}&resource={4}&prompt=select_account')
+    url = url.format(authority_url, _CLIENT_ID, reply_url, 'code', resource)
```

Comment:
Is there an API in `adal` to construct this URL?

Reply:
no